elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -133,6 +136,7 @@ $variables: (
   &--wrapper {
     display: block; 
     white-space: nowrap;
+    min-width: calc(100%);
```

Comment:
@starryeyez024 Why do we need the calc function?

Reply:
ooh we definitely dont, i'll remove that